config/translations/lxqt-config-globalkeyshortcuts_fr.desktop

```diff
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Type=Application
```

Comment:
I am sorry. 
I am not still used of GitHub nor rules of translation. 
But it will serve me afterward.